util/build_prep/centos/prep.sh.in

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env bash
+
+# -----BEGIN COMMON.SH-----
+# -----END COMMON.SH-----
+
+yes | yum update -y
+
+yes | yum install -y git wget openssl bzip2; # <boost>
+
+yes | yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm; # epel 7
+yes | yum install -y jq || exit 1
+
+yes | yum install -y rpm-build || exit 1
+yes | yum install -y glibc-devel glibc-headers make which libstdc++-static || exit 1
+yes | yum install -y centos-release-scl || exit 1
+yes | yum install -y llvm-toolset-7-cmake devtoolset-7-llvm|| exit 1
+
+# Ensure we have a new enough Boost
+(
+	eval "$(scl enable llvm-toolset-7 devtoolset-7 "bash -c 'set | grep ^PATH='")"
+	if ! have boost; then
+		bootstrap_boost -m -k
```

Comment:
is there a reason for keeping the archive file (`-k` option)?

Reply:
just a left over, thanks for catching it